Makefile

```diff
@@ -503,6 +503,10 @@ vendor/check/prow:
 vendor/fix:
 	go mod tidy
 	go mod vendor
+	cd test
+	go mod tidy
```

Comment:
I'm interestd to get thoughts on whether we should vendor the test folders dependencies. The goal of vendoring is about rebuilding older versions. W

Reply:
I've no real preference on this. Probably better to not vendor the test directory as it would otherwise not reduce the amount of files committed 🤔